src/Console/ModelsCommand.php

```diff
@@ -886,6 +888,15 @@ protected function getCollectionClass($className)
         return '\\' . get_class($model->newCollection());
     }
 
+    /**
+     * Returns the available relation types
+     */
+    public function getRelationTypes(): array
```

Comment:
I don't think it needs to be `public` since the basic feature is already configurable?
```suggestion
    protected function getRelationTypes(): array
```